pkg/openstack/loadbalancer.go

```diff
@@ -1433,7 +1433,7 @@ func (lbaas *LbaasV2) buildListenerCreateOpt(port corev1.ServicePort, svcConf *s
 }
 
 // getMemberSubnetID gets the configured member-subnet-id from the different possible sources.
-func (lbaas *LbaasV2) getMemberSubnetID(service *corev1.Service, svcConf *serviceConfig) (string, error) {
+func (lbaas *LbaasV2) getMemberSubnetID(service *corev1.Service) (string, error) {
```

Comment:
I'll defer to the maintainers to chime in on this change; `svcConf` is indeed unused, but I don't know why it was there in the first place.

CC @jichenjc @kayrus

Reply:
I've took a quick look at the `git blame`, looks like it was there by mistake: